webapp/src/Twig/TwigExtension.php

```diff
@@ -536,9 +541,15 @@ public function printResult(?string $result, bool $valid = true, bool $jury = fa
                 break;
             case 'correct':
                 $style = 'sol_correct';
+                if ($onlyAcceptedAndRejected) {
+                    $result = 'accepted';
+                }
```

Comment:
Why this specific case? Accept -> Correct is no change.

Reply:
@meisterT what do you think?

Reply:
So this would get to:

Correct vs {Wrong answer, Timelimit etc.}
Accepted vs Rejected

I prefer limiting the verdict names but I get the code now
So I propose
Correct vs Incorrect just as alternative.